### ISSUE TITLE
Have weighting curves been validated?

### ISSUE THREAD

**Issue by sammlapp** [endolith/waveform_analysis#25]
The README mentions that some parts of this repo have not been validated. This seems to be the best Python resource for A, C weighting of digital audio, and it would be useful if someone could confirm that the implementation is correct.

**Comment by endolith** [comment]
Validated in what way?   I have scripts to compare it with the limits specified in the standards, and it meets them for a range of sample rates.  Not sure if I put them on here or not.

Yes, the repo is disorganized and unfinished and I'm sorry about that.  Ironically it's my most popular repo and also the worst.  If more people bother me about it I'll prioritize cleaning it up.

**Comment by endolith** [comment]
In fact I started it https://github.com/endolith/weighting_filters

**Comment by endolith** [comment]
See https://github.com/berndporr/sound_weighting_filters instead

**Comment by sammlapp** [comment]
sorry for delay in replying, but thank you for that resource  - it's certainly useful to me!

**Comment by jonnor** [comment]
The MZTi method is not included in the berndporr repository, nor in this one? And according to the graphs showed in the linked comments, that is the one that performed the best?

**Comment by endolith** [comment]
@jonnor Yeah it should probably be adopted and just document the discrepancies at low fs
